tests/functional/autocomplete/test_completer.py

```diff
@@ -236,6 +236,11 @@ def test_autoclose_brackets(self):
         self.assertEqual(suggestions[-1].name,
                          'key={M=[key1={M=[key2={BS=[1]}]}]}')
 
+    def test_not_start_autocompletion_wo_trailing_space(self):
+        parsed = self.parser.parse('aws dynamodb put-item --item')
```

Comment:
I don't think the `--items` parameter even has auto-completions? Maybe we should use the `codebuild create-project` command from before and just assert it does not have auto-completions without the whitespace.

Reply:
Or I guess initially it will auto-complete eventually but not on the first input character to the value because this value is a map.

Reply:
it has, basically all the other testcases use it

Reply:
Or to rephrase it, the string `aws dynamodb put-item --item` will not trigger the bug. We should use a string that does trigger the bug to make sure we have a case that catches it in the future.

Reply:
Ah, yeah, makes sense